__docs__/wonder-blocks-icon/accessibility.mdx

```diff
@@ -13,19 +13,19 @@ import * as AccessibilityStories from './accessibility.stories';
 * The color contrast should pass WCAG.
   * At least 3:1 contrast ratio for icons.
 * Ideally, an icon should be accompanied by text. However, if an icon
-  is not accompanied by text, provide alt text.
-  * Do not provide alt text for decorative icons. Set the alt text
-    to an empty string (`""`).
+  is not accompanied by text, provide an `aria-label`.
+  * If you are using `aria-label`, make sure to also use `role="img"` so that
+    screen readers know that the element is an image and not just decorative
+    text.
+  * If the icon is decorative, set `aria-hidden` to `true` and do not provide an
+   `aria-label`. This will hide the icon from screen readers.
   * The `title` attribute is often used to display a description on mouse
     hover, but please note that use of the `title` attribute is discouraged
     as titles are not available to keyboard-only or touch-only users.
     More information can be found in the [References](#references) below.
 * Icons should go on the opposite side as left-to-right languages
   for right-to-left languages.
   * The icon may also need to be mirrored.
```

Comment:
For my own curiosity, when should icons be mirrored? And how should developers do this?

Reply:
🤔 I never thought about this.... my take would be to use `rotate(180deg)`, but not sure if it needs to be mirrored in a different way (cc @nishasy in case you have any thoughts about this).